src/versions/versions.controller.ts

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import {
+  Controller,
+  Get,
+  HttpStatus,
+  Post,
+  Query,
+  UseGuards,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { ApiExcludeEndpoint, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { ApiKeyGuard } from '../auth/guards/api-key.guard';
+import { CreateVersionDto } from './dto/create-version.dto';
+import { SerializedVersion } from './interfaces/serialized-version';
+import { serializedVersionFromRecord } from './utils/version-translator';
+import { VersionsService } from './versions.service';
+
+@ApiTags('Versions')
+@Controller('versions')
+export class VersionsController {
+  constructor(private readonly versionsService: VersionsService) {}
+
+  @ApiOperation({ summary: 'Gets the version of the Iron Fish package' })
+  @Get()
+  async version(): Promise<Record<string, SerializedVersion>> {
+    const latestVersion = await this.versionsService.getLatest();
+    if (latestVersion === null) {
+      return {};
+    }
+    return {
+      ironfish: serializedVersionFromRecord(latestVersion),
```

Comment:
I like that we're returning an object now instead of the version string, but since I don't think we'll need the `created_at` field, what do you think about just returning the version directly?
```suggestion
      ironfish: latestVersion.version,
```

Reply:
I'm leaning towards continuing to return an object. After you got me thinking about the shape of the return data, I felt like returning an object here makes it less of a breaking change down the line if we want to add/remove fields. Could be useful to see when a version was deployed, also. I'm not sure if there's an immediate use for it, but I could see that being useful to surface alongside the version when we start consuming this endpoint